ArduPlane/system.cpp

```diff
@@ -230,6 +230,28 @@ bool Plane::set_mode(Mode &new_mode, const ModeReason reason)
     Mode &old_mode = *control_mode;
     const ModeReason previous_mode_reason_backup = previous_mode_reason;
 
+    // cancel inverted flight
+    auto_state.inverted_flight = false;
+
+    // don't cross-track when starting a mission
+    auto_state.next_wp_crosstrack = false;
```

Comment:
why add this?

Reply:
again, wasn't me who wrote this line of code, will remove.